ui/src/settings.vue

```diff
@@ -537,13 +560,55 @@ export default {
             },
             passwordSuggestions: [],
             passwordWarning: "" ,
-           
+            userEditJSON : null,
+            scopeModel : {
+                "brainlife": {
+                    "user" : false,
+                    "admin" : false,
+                },
+                "warehouse": {
+                    "admin" : false,
+                    "datatype.create" : false
+                },
+                "amaretti": {
+                    "admin" : false,
+                    "resource.create" : false,
+                },
+                "auth": {
+                    "admin" : false,
+                },
+            },
+            tabs: [
+                {id: 'profile', label: "Profile"},
+                {id: 'account', label: "Account"},
+                {id: 'notification', label: "Notification"},
+                {id: "users", label: "Users"},
+                {id: "groups", label: "Groups"},
+            ],
+
+            permissionScope: {
+                "brainlife": {
+                    "user" : "description",
+                    "admin" : "description of admin"
+                },
+                "warehouse": {
+                    "admin" : "description of admin of warehouse",
+                    "datatype.create" : "description of datatype.create in warehouse"
+                },
+                "amaretti": {
+                    "admin" : "description of admin of amaretti",
+                    "resource.create" : "description of resource.create in amaretti"
+                },
+                "auth": {
+                    "admin" : "description of admin of auth"
```

Comment:
for description "Allows user to perform administrative tasks on authorization service"